modules/api/src/main/java/net/countercraft/movecraft/craft/Craft.java

```diff
@@ -322,25 +324,25 @@ public int getTickCooldown() {
 
         int chestPenalty = (int)((materials.get(Material.CHEST) + materials.get(Material.TRAPPED_CHEST)) * type.getChestPenalty());
         if(!cruising)
-            return type.getTickCooldown(w) + chestPenalty;
+            return (type.getTickCooldown(w) * currentGear) + chestPenalty;

```

Comment:
This will increase the tickCooldown due to the type config, but not due to the chestPenalty.  For example, a craft with a tickCooldown from the type as 3, but a chestPenalty of 2, will go from 5 to 8 tickCooldown from gear 1 to gear 2.